lib/auth/auth_with_roles.go

```diff
@@ -3771,15 +3768,15 @@ func (a *ServerWithRoles) SignDatabaseCSR(ctx context.Context, req *proto.Databa
 //
 // This certificate can be requested by:
 //
-//  - Cluster administrator using "tctl auth sign --format=db" command locally
-//    on the auth server to produce a certificate for configuring a self-hosted
-//    database.
-//  - Remote user using "tctl auth sign --format=db" command with a remote
-//    proxy (e.g. Teleport Cloud), as long as they can impersonate system
-//    role Db.
-//  - Database service when initiating connection to a database instance to
-//    produce a client certificate.
-//  - Proxy service when generating mTLS files to a database
+//   - Cluster administrator using "tctl auth sign --format=db" command locally
```

Comment:
From where do these formatting changes come from?

Reply:
Go 1.19 doc comments: https://tip.golang.org/doc/comment

Reply:
(This happens on save with gofmt/gopls now, so you'll start seeing these until we get to all the ones that aren't formatted)

Probably a good idea to open a separate PR after this and reformat the entire codebase so we stop seeing these little diffs in unrelated PRs.